frontend/src/components/molecules/FacetedSearchGroup/FacetedSearchGroup.js

```diff
@@ -3,6 +3,7 @@ import {
   Checkbox, Collapse, Form, Typography, Space,
 } from 'antd';
 import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
I'm not sure why you are adding keys using uuidv4 considering it caused other bug?

Reply:
The pull request is to fix the bug caused by your change in the SearchForm. You were generating hundreds of similar keys in the Option loop (line ~105)

Reply:
I will remove uuid from FacetedSearchGRoup and update the PR

Reply:
Uuids are removed from the FacetedSearchGroup